controllers/vspheremachine_controller.go

```diff
@@ -664,15 +664,26 @@ func (r machineReconciler) waitReadyState(ctx *context.MachineContext, vm *unstr
 }
 
 func (r *machineReconciler) clusterToVSphereMachines(a client.Object) []reconcile.Request {
-	requests := []reconcile.Request{}
+	log := r.Logger
+	var requests []reconcile.Request
 	machines, err := infrautilv1.GetMachinesInCluster(goctx.Background(), r.Client, a.GetNamespace(), a.GetName())
```

Comment:
Checkout the method `GetVSphereMachinesInCluster` in the same package `cluster-api-provider-vsphere/pkg/util`. You can use that to queue all the VSphereMachine requests.

Reply:
oh cool